### PR TITLE
Deploy script

### PR DESCRIPTION
**Pull request by stuartsan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,9 @@
     "test": "",
     "build": "npm run clean; webpack --env=prod --colors",
     "clean": "rm -rf dist",
-    "dist:server": "pushd dist; python -m SimpleHTTPServer; popd"
+    "dist:server": "pushd dist; python -m SimpleHTTPServer; popd",
+    "dev-deploy": "aws s3 sync dist/ s3://mapstaging",
```
Comment (stuartsan) [inline package.json:12]: I gave these different names to make it a little harder to accidentally deploy prod :grin:

File: README.md [hunk 2]
```diff
@@ -25,4 +25,21 @@ npm run dist:server
...
 
 ## Deployment
-TBD
+
+First build the app per instructions in the previous step, serve it locally, and make sure things look good.
+
+Then do either:
+
+```
+npm run dev-deploy
```
Comment (shazow) [inline README.md]: yarn it up! :)

**Comment by shazow** [comment]
~Any reason why we're deploying prod to a docker rather than s3+route53 or somesuch?~

File: README.md (hunk 2 above)
Comment (stuartsan) [reply README.md]: Will do! Also noticed a typo in one of these commands. Also, will see if we know the bucket name for prod yet...

**Comment by shazow** [comment]
Ah I think I misread the dist server (for testing a prod-like deployment?) as deployment.

**Comment by stuartsan** [comment]
This PR https://github.com/aclu-people-power/map/pull/57/files may answer your Q -- we are deploying prod to s3 but I don't know the bucket name yet, and the docker thing is just to serve the production stuff locally, soon-to-be swapped in when running `yarn run dist:server`.

**Pull request merged**